public_html/wp-content/themes/campsite-2017/template-parts/content.php

```diff
@@ -15,6 +15,12 @@
 	<header class="entry-header">
 		<?php
 
+		if ( ( ! is_single() || is_singular( 'post' ) ) && has_post_thumbnail() ): ?>
```

Comment:
This is where the second feature flag would need to go, assuming that Jetpack content options do apply to custom post types as well. I think it would be something like

```suggestion
		if ( wcorg_skip_feature( 'cs17_display_cpt_featured_image' ) ) {
			$show_featured_image = ( ! is_single() || is_singular( 'post' ) ) && has_post_thumbnail();
		} else {
			$show_featured_image = has_post_thumbnail();
		}

		if ( $show_featured_image ) : ?>
```

Reply:
We won;t need this as per https://github.com/WordPress/wordcamp.org/pull/111#discussion_r281394415. Resolving